cloudify_cli/utils.py

```diff
@@ -676,5 +712,5 @@ def validate_definitions_version(self):
 
 def build_manager_host_string(user='', ip=''):
     user = user or get_management_user()
-    ip = ip or get_management_server_ip()
+    ip = ip or get_rest_host()
```

Comment:
leave it get_management_server_ip in this case as this function is only used by ssh related functions.